### DISCUSSION TITLE
Slide uploading.

### DISCUSSION THREAD

**Discussion by adarsh7raj** [camicroscope/caMicroscope#674] [Q&A]
I am not able to upload the slides which i downloaded, can anyone help me what is WSI file?

**Comment by birm** [comment]
wsi stands for Whole Slide Image. What files did you download and where did you get them?

**Comment by adarsh7raj** [comment]
First i went to caMicroscope.org then to live demo then clicked on the home button then this page arrives
(http://localhost:4010/apps/table.html)

<img width="957" alt="Screenshot 2024-03-02 221117" src="https://github.com/camicroscope/caMicroscope/assets/130682848/11fd98f8-f53d-4158-abf7-9b3eefa1bec5">
On the locally hosted caMIcroscope i went to slides section i.e:http://localhost:4010/apps/table.html

and when i tried to upload a slide then  this error occurs:

https://github.com/camicroscope/caMicroscope/assets/130682848/8191241e-8721-4ec2-abb4-3a23cc60f774

the finish upload button does not  appears and the slide does not get added.

**Reply by barchakuz** [reply]
make sure your ca-loading is working fine in docker
